torch/distributed/fsdp/fully_sharded_data_parallel.py

```diff
@@ -87,29 +95,34 @@ class TrainingState_(Enum):
 class FullyShardedDataParallel(nn.Module):
     """
     A wrapper for sharding Module parameters across data parallel workers. This
-    is inspired by `Xu et al.`_ as well as the ZeRO Stage 3 from DeepSpeed_.
+    is inspired by `Xu et al.`_ as well as the ZeRO Stage 3 from `DeepSpeed`_.
     ``FullyShardedDataParallel`` is commonly shorten to FSDP.
-    .. _`Xu et al.`: https://arxiv.org/abs/2004.13336
-    .. _DeepSpeed: https://www.deepspeed.ai/
+    `Xu et al.`_: https://arxiv.org/abs/2004.13336
+    `DeepSpeed`_: https://www.deepspeed.ai/
```

Comment:
Not sure the new version will work or not. But it seems the old one might be working, expect that we are missing one new line before `Xu et al.`. If this still doesn't work, we can make the reference explicit `text <link>`__

<img width="660" alt="Screen Shot 2022-02-01 at 3 20 38 PM" src="https://user-images.githubusercontent.com/16999635/152044876-bf122b8c-a5ec-4f71-9599-66b857978f5a.png">